visualization/app/codeCharta/ui/customConfigs/customConfigs.component.scss

```diff
@@ -1,9 +1,15 @@
 .custom-configs {
```

Comment:
We can remove the empty selector (as suggested by [sonar's detected code smell](https://sonarcloud.io/project/issues?id=maibornwolff-gmbh_codecharta_visualization&pullRequest=2736&resolved=false&types=CODE_SMELL))